src/languages/php.js

```diff
@@ -379,12 +436,25 @@ export default function(hljs) {
       },
       VARIABLE,
       FUNCTION_INVOKE,
+      LEFT_AND_RIGHT_SIDE_OF_DOUBLE_COLON,
+      {
+        match: [
+          /const/,
+          regex.concat(WHITESPACE, "+"),
```

Comment:
This is super verbose please use `\s` when you can, and only fall back to WHITESPACE when there are issues.

Reply:
> This is super verbose please use `\s` when you can, and only fall back to WHITESPACE when there are issues.

Yes, it is. I never saw it (nobody would do that), but the language allows to start the const name with [non-breaking](http://sandbox.onlinephpfunctions.com/code/9780e686f54fad41c69cf310500490cb6eec1384) space. I wanted to use the same rule for whitespace everywhere.  
Also, there is a case of `Polynomial backtracking` (with \s+) and some code formatting rules might have multiple spaces:  
```
const FOO                       = 1;
const SOMETHING                 = 2;

Reply:
>  but the language allows to start the const name with non-breaking space

I'm not sure we care about this or this matter since the highlighting for all intents and purposes is identical.

> Also, there is a case of Polynomial backtracking (with \s+) 

These can usually be fixed (and you can leave it to me if you want).  So again, let's use `\s` (when it works, which is 99% of the time) and only fall back to more complex usages when it fails to work.